### PR TITLE
Added new  task to verify  that openebs is removed from cluster.

### PR DESCRIPTION
**Pull request by amitbhatt818**
- Added new task to verify openebs is removed or not from cluster.

Signed-off-by: Amit Bhatt <amitbhatt818@gmail.com>

### CODE REVIEW

**Comment by avishnu** [comment]
Is this fixing https://github.com/mayadata-io/oep-e2e/issues/303 ?

**Comment by amitbhatt818** [comment]
> Is this fixing #303 ?

yes @avishnu

File: utils/openebs-cleanup.yml [hunk 1]
```diff
@@ -57,4 +57,33 @@
...
+    method: GET
+    url_username: "{{ username.stdout }}"
+    url_password: "{{ password.stdout }}" 
+    return_content: yes
+    force_basic_auth: yes
+    body_format: json
+  register: openebs_details
+  until: "{{ openebs_details.json.data[0].state}} =='Removed'"
+  retries: 10
+  delay: 30
```
Comment (harshshekhar15) [inline utils/openebs-cleanup.yml]: It's better to increase the retries and decrease the delay. Set delay to 10 and retries to 30.

File: utils/openebs-cleanup.yml [hunk 2]
```diff
@@ -57,4 +57,33 @@
...
+    url: "{{ director_url }}/v3/clusters/{{ cluster_id }}"
+    method: GET
+    url_username: "{{ username.stdout }}"
+    url_password: "{{ password.stdout }}"
+    return_content: yes
+    force_basic_auth: yes
+    body_format: json
+  register: openebs_components_details
+     
+- name: Fetching specific component deatils to verify
```
Comment (harshshekhar15) [inline utils/openebs-cleanup.yml]: typo - `details`

File: utils/openebs-cleanup.yml [hunk 3]
```diff
@@ -57,4 +57,33 @@
...
+  set_fact:
+    component_details: "{{ openebs_components_details.json.clusterComponents}}"
```
Comment (harshshekhar15) [inline utils/openebs-cleanup.yml]: Add a space after `clusterComponents`

**Pull request merged**